src/zoid/card-fields/component.jsx

```diff
@@ -150,6 +151,17 @@ export const getCardFieldsComponent : () => CardFieldsComponent = memoize(() : C
             },
 
             props: {
+                action: {
+                    type: 'object',
+                    value: ({props}) => {
+                        if (props.action) {
+                            return props.action
+                        } else {
+                            return props.parent.props.action
```

Comment:
Did we sort out why this was going on or is it kinda hand-wavy "zoid stuff sometimes"?

Reply:
this is because we don't have a "leader" or "facilitator" iframe with Zoid. We define the parent component but we don't actually render it so it never creates an iframe. I think the "leader" iframe becomes a random child component that does have access to its parent. Its really weird and we should really re-think using Zoid for CardFields once we have time to breathe